server/sonar-web/src/main/js/app/components/GlobalFooter.js

```diff
@@ -36,8 +36,11 @@ class GlobalFooter extends React.Component {
               Embedded database should be used for evaluation purpose only
             </p>
             <p>
-              The embedded database will not scale, it will not support upgrading to newer versions of SonarQube,
-              and there is no support for migrating your data out of it into a different database engine.
+              The embedded database will not scale, it will not support upgrading to newer
+              {' '}
```

Comment:
Is this added automatically when wrapping the text to 100 characters ? We don't need this right ?